### DISCUSSION TITLE
Does CefSharp support headers in the WebSocket (WS/WSS) handshake?

### DISCUSSION THREAD

**Discussion by TamirHCL** [cefsharp/CefSharp#4724] [Q&A]
According to https://stackoverflow.com/questions/71522310/websocket-not-working-in-cefsharp-project#comment126413464_71522310 CefSharp does not support headers in the WebSocket (WS/WSS) handshake. 
Is this still correct today in the latest versions of CefSharp?

**Comment by campersau** [comment]
It is still not supported by the browser websocket spec: https://github.com/whatwg/websockets/issues/16
